src/Plot.js

```diff
@@ -355,7 +355,7 @@ export default class Plot extends Viz {
         }
 
         // set axis title if not discrete
-        if (str !== k && (!this[`_${k}Config`].title || this[`_${k}Title`] === this[`_${k}Config`].title) && this._discrete !== k) {
+        if (str !== k && (!this[`_${k}Config`].title === undefined || this[`_${k}Title`] === this[`_${k}Config`].title) && this._discrete !== k) {
```

Comment:
Ok, don't kill me, but I think this solution may be even EASIER than we thought.

I saw that you included an exclamation point before the undefined conditional, which I didn't expect, so I pulled this branch and did some testing.

...not only does it appear that the exclamation point doesn't matter, but the whole undefined condition doesn't seem to do anything and it all just works without it!? Try removing `!this[`_${k}Config`].title === undefined || ` from this code and see if that works for you as well 😅

Reply:
I did a few test removing ```!this[_${k}Config].title === undefined ||``` from code and works pretty well.
Let me know if you find a border case that escaped my testing so I can review it 💪 .